tools/internal_ci/linux/grpc_xds_bazel_python_test_in_docker.sh

```diff
@@ -48,12 +48,14 @@ touch "$TOOLS_DIR"/src/proto/grpc/testing/__init__.py
 
 bazel build //src/python/grpcio_tests/tests_py3_only/interop:xds_interop_client
 
+# Test cases "path_matching" and "header_matching" are not included in "all",
+# because not all interop clients in all languages support these new tests.
```

Comment:
It wasn't done that way because of the reason mentioned here. But now all languages support the new tests, so we can add the two cases to `all`.

Let's do this as is, and do the cleanup separately.